localstack/ext/java/src/test/java/cloud/localstack/S3UploadTest.java

```diff
@@ -49,6 +50,30 @@ public void testGreaterThan128k() throws Exception  {
 	@Test
 	public void testLessThan128k() throws Exception  {
 		testUpload(String.join("", Collections.nCopies(13107, "abcdefghij"))); // Just slightly less than 2^17 bytes
+    }
+
+	/**
+	 * Tests upload of empty file. This is an operation that hadoop's S3AFilesystem executes to create "directories"
+	 * in S3.
+	 *
+	 * This test is currently failing because the S3 server doesn't properly calculate the MD5 of null string.
+	 */
+	@Test
```

Comment:
I'm happy to throw an `@Ignored` on here to make the CI pass. I mainly wanted to explicitly bring this issue up in a PR since it seems like there are a number of open issues around this bug.